json_configs/template.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * configuration.cpp
+ *
+ * Copyright (C) 2021-2025 Daniel Horn, Roy Falk, ministerofinformation,
+ * David Wales, Stephen G. Tuggy, and other Vega Strike contributors
+ *
+ * https://github.com/vegastrike/Vega-Strike-Engine-Source
+ *
+ * This file is part of Vega Strike.
+ *
+ * Vega Strike is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Vega Strike is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+#include <iostream>
+#include <exception>
+#include <boost/json.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#include "configuration.h"
+
+#ifndef _USE_MATH_DEFINES
+#define _USE_MATH_DEFINES
+#endif
+#include <math.h>
+#include <vs_logging.h>
```

Comment:
I don't see where you're using logs here.

Reply:
I wasn't. I meant to change the `std::cerr << ...`calls to VS_LOG calls, but I had forgotten to finish that. Should be correct now.